rpm/phonesim.spec

```diff
@@ -48,24 +36,13 @@ FallbackTimeservers.
 %prep
 %setup -q -n %{name}-%{version}
 
-# >> setup
-# << setup
-
 %build
-# >> build pre
-# << build pre
 
```

Comment:
I would skip this empty line too. Maybe also the next empty line as well.

Reply:
Sure.